cmd/feed-ingress/main.go

```diff
@@ -136,3 +143,8 @@ func createLB() types.LoadBalancer {
 		DefaultAllow:      ingressAllow,
 	})
 }
+
+func startMetrics() {
+	http.Handle("/metrics", prometheus.Handler())
+	http.ListenAndServe(metricsListenAddress, nil)
```

Comment:
I'm not sure you can have multiple of these, I think they use global state (so it will conflict with our /health /debug one).